src/components/PageBlocks/Configurator.vue

```diff
@@ -80,14 +81,38 @@
           />
         </b-form-group>
 
-        <b-form-checkbox
-          v-model="block.style.wrap.kind"
-          value="card"
-          unchecked-value="plain"
-          switch
+        <b-form-group>
+          <b-form-checkbox
+            v-model="block.style.wrap.kind"
+            value="card"
+            unchecked-value="plain"
+            switch
+          >
+            {{ $t('general.wrap') }}
+          </b-form-checkbox>
+        </b-form-group>
+
+        <b-form-group
+          v-if="block.options.refreshRate !== undefined"
+          :label="$t('general.refresh.label')"
+          :description="$t('general.refresh.description')"
         >
-          {{ $t('general.wrap') }}
-        </b-form-checkbox>
+          <b-col
+            cols="12"
+            sm="3"
+            class="pl-0"
+          >
+            <b-input-group append="s">
+              <b-form-input
+                v-model="block.options.refreshRate"
+                type="number"
+                number
+                min="0"
+                @blur="updateRefresh"
```

Comment:
I think that instead of on blur you can do this with a computed with a get and set.

And just use v-model. Try it please